KMB/make_KMB_info.py

```diff
@@ -706,20 +752,18 @@ def get_depicted_place(self):
         kommun_map = self.kmb_info.mappings['kommun']
         socken_map = self.kmb_info.mappings['socken']
         depicted_place = None
-        if not self.land or self.land == 'se':
+        if not self.land or self.land == 'SE':
             if 'Gotland' in (self.lan, self.landskap) and not self.kommun:
                 # since lan/landskap/kommun are equivalent in this case
                 self.kommun = '0980'  # Gotlands kommun
```

Comment:
Could this be retrieved from some mapping file instead of being a hardcoded magic number? :)